secp256k1.go

```diff
@@ -44,8 +44,12 @@ func init() {
 	}
 }
 
-// HashSize of array used to store hashes. See Hash.
-const HashSize = 32
+const (
+	// HashSize of array used to store hashes. See Hash.
+	HashSize = 32
+	// SerializedPrivateKeySize defines the length in bytes of SerializedPrivateKey
```

Comment:
Please add a blank line above this line.

Reply:
+1
Would've thought `go fmt` will do that for me.